R/snapshot-auto.R

```diff
@@ -64,18 +69,14 @@ renv_snapshot_auto_enabled <- function(project) {
 
 }
 
-renv_snapshot_auto_update <- function(project) {
+renv_snapshot_auto_update <- function(project = renv_project_get() ) {
```

Comment:
Would it be worth having this function return a boolean value indicating if the lockfile was updated / changed? (basically thinking of if we should move the `old != new` check a bit further inside the call stack)

Reply:
I made `renv_snapshot_auto_impl()` return the logical; I think that's what you were thinking.